### PR TITLE
Update RF24Ethernet Docs: config&setup

### PR DESCRIPTION
**Pull request by TMRh20**
- Link to new installer
- Modify commands for CMAKE
- Add commands to manually configure the interface when not running as root
- Remove step 4 for Arduino and clarify

I will continue reviewing the docs and add to this PR as necessary.

### CODE REVIEW

**Comment by 2bndy5** [comment]
Nice work on the new video!

I noticed some build warnings in the SSH terminal. IIRC, the warnings are specific to the ncurses example, but I never got around to addressing those...

**Comment by TMRh20** [comment]
Thanks! 
Yeah, the warnings affect RF24Mesh and Ethernet, its the topology drawing piece.

**Comment by 2bndy5** [comment]
Just a reminder: You can publish doc changes without publishing a release by [manually triggering the workfklow](https://github.com/nRF24/RF24Ethernet/actions/workflows/doxygen.yml)

File: docs/config_setup.md [hunk 1]
```diff
@@ -5,33 +5,37 @@
...
    ./install.sh
    ```
 2. Next, build and run the [RF24Gateway_ncurses example](https://nrf24.github.io/RF24Gateway/RF24Gateway_ncurses_8cpp-example.html)
    ```shell
-   cd RF24Gateway/examples/ncurses
+   cd rf24libs/RF24Gateway/examples/build
    make
-   sudo ./RF24Gateway_ncurses
+   cd ncurses
+   ./RF24Gateway_ncurses
```
Comment (2bndy5) [inline docs/config_setup.md:23]: no more sudo?

**Review by 2bndy5 — APPROVED** [review]

File: docs/config_setup.md (hunk 1 above)
Comment (TMRh20) [reply docs/config_setup.md:23]: nope, not needed with the defaults. Don't want to encourage people to run as root if they don't have to.

**Pull request merged**